### DISCUSSION TITLE
How to turn off annoying underlines in markdown?

### DISCUSSION THREAD

**Discussion by imroc** [LazyVim/LazyVim#1470] [Q&A]
A lot of underlines are automatically added, which increases reading difficulties:
![image](https://github.com/LazyVim/LazyVim/assets/7448852/78e60a9f-ae2d-4624-bba1-372f015a3f71)

**Comment by imroc** [comment]
cc @folke

**Comment by imroc** [comment]
When I use [LunarVim](https://www.lunarvim.org/), there is no underlines:
![image](https://github.com/LazyVim/LazyVim/assets/7448852/e15f3a70-ee34-49ed-80a1-766c92501eab)

**Comment by imroc** [comment]
I found it's because the default autocmd of LazyVim: https://github.com/LazyVim/LazyVim/blob/v6.3.0/lua/lazyvim/config/autocmds.lua#L78

Which turns on the spell and wrap, and In Chinese, neovim cannot identify most of the words, this results in ugly underlines filling the screen, seriously affecting the reading experience:

![image](https://github.com/LazyVim/LazyVim/assets/7448852/e6594b95-9c07-43dc-bd23-62bade098645)

I tried overriding it in ftplugin, but it didn’t work. It seems that ftplugin is executed before the default autocmd.

Now I temporarily use a custom autocmd to override and avoid the ugly underline.

![image](https://github.com/LazyVim/LazyVim/assets/7448852/454aba6e-5843-4b2d-9ef9-cf73216e1b48)

It is hoped that LazyVim will support the option of disabling default spell/wrap in the future.

**Reply by dpetka2001** [reply]
There are currently keybindings to disable both of them, `<leader>us` for spelling and `<leader>uw` for wrapping. If you want the default behavior to be disabled by default, then an autocmd is the correct route to achieve that. Instead of writing a whole `autocmd` yourself, you could also do `vim.api.nvim_create_augroup("lazyvim_wrap_spell", { clear = true })` in your `/lua/config/autocmds.lua`, so that you just disable the default Lazyvim autocmd.

**Reply by imroc** [reply]
Nice catch, thanks

**Reply by grybouilli** [reply]
is it possible rather than to disable the spelling errors, to choose the language?
